src/vaishnaviVJun20_arrayExamples/Armstrong.java

```diff
@@ -0,0 +1,27 @@
+package vaishnaviVJun20_arrayExamples;
+
+//WAP to check if a number is Armstrong or not.
+
+public class Armstrong {
+	
+	void findIfNumberIsArmstrong(int input) {
+		int Originalnumber = input;
+		int sum =0;
+		while(input>0) {
+			int num = input%10;
+			input = input/10;
+			int result = num*num*num;
+			sum += result;
```

Comment:
On this line you can directly formulate the value of variable sum.
No need to declare extra variables and store values separately.(remove line 11 and 13)

Reply:
--Done